tests/agent-overrides/manifests/a00-rbac.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
i think some of the other tests just apply the tenant chart now to avoid having to do rbac each time